tests/foreman/cli/test_computeresource_azurerm.py

```diff
@@ -556,3 +556,133 @@ def test_positive_azurerm_host_provisioned(
         # Azure cloud
         assert self.hostname.lower() == azureclient_host.name
         assert self.vm_size == azureclient_host.type
+
+
+@pytest.mark.run_in_one_thread
+class TestAzureRMBYOSFinishTemplateProvisioning:
```

Comment:
Same as above question for Azure Tests! This has been deleted why we are adding again ?

CC @adarshdubey-star !